<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveStateDialog</class>
 <widget class="QDialog" name="SaveStateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnSlot1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Slot 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSlot2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Slot 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSlot3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Slot 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSlot1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>105</y>
     <width>160</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSlot2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>145</y>
     <width>160</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSlot3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>185</y>
     <width>160</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-Choose a slot to save to-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
